tests/http/kvmagent/test_callback_storage.py

```
@@ -0,0 +1,163 @@
+        _skip_if_not_loaded(response, '/localstorage/volume/getbaseimage')
+        assert response.status_code == 200
+
+        result = async_callback.wait(response.task_uuid, timeout=15.0)
+        assert isinstance(result, dict), "callback should return a dict"
+
+    def test_get_qcow2_reference(self, kvmagent_client, async_callback):
+        """Test /localstorage/getqcow2reference returns reference info."""
+        callback_url = async_callback.get_callback_url()
+        response = kvmagent_client.post(
+            '/localstorage/getqcow2reference',
+            data={'path': '/nonexistent', 'searchingDir': '/tmp'},
```

Ruff (0.15.2)

[error] Probable insecure usage of temporary file or directory: "/tmp"

(S108)